packages/guilded.js/lib/gateway/handler/TeamMemberEventHandler.ts

```diff
@@ -22,13 +23,13 @@ export class TeamMemberEventHandler extends GatewayEventHandler {
         const newMember = new Member(this.client, { ...data.d.member, serverId: data.d.serverId, id: data.d.member.user.id });
         const newUser = new User(this.client, data.d.member.user);
 
-        this.client.members.cache.set(TeamMemberEventHandler.buildMemberKey(data.d.serverId, data.d.member.user.id), newMember);
+        this.client.members.cache.set(buildMemberKey(data.d.serverId, data.d.member.user.id), newMember);
         this.client.users.cache.set(newUser.id, newUser);
         return this.client.emit(constants.clientEvents.TEAM_MEMBER_JOINED, newMember);
     }
     teamMemberRemoved(data: WSTeamMemberRemovedPayload): boolean {
-        const existingMember = this.client.members.cache.get(TeamMemberEventHandler.buildMemberKey(data.d.serverId, data.d.userId));
-        if (this.client.options.cache?.removeMemberOnLeave) this.client.members.cache.delete(TeamMemberEventHandler.buildMemberKey(data.d.serverId, data.d.userId));
+        const existingMember = this.client.members.cache.get(buildMemberKey(data.d.serverId, data.d.userId));
```

Comment:
Can condense this line’s key and the next line’s key into the next line with a variable